components/concurrency_manager/src/lib.rs

```diff
@@ -278,29 +284,28 @@ impl ConcurrencyManager {
         limit: TimeStamp,
         source: impl slog::Value + Display,
         using_approximate: bool,
+        tso_confirmed: bool,
     ) -> Result<(), InvalidMaxTsUpdate> {
-        let can_panic = !using_approximate;
         error!("invalid max_ts update";
```

Comment:
The `WARN` log level could be used if the error is not confirmed. Usually `WARN` indicates there could be errors, and `ERROR` unexpected error already happens and the system may not run in a normal state.